### ISSUE TITLE
what facial segment model have you used for inpainting mask?

### ISSUE THREAD

**Issue by YuGuilliman** [ICTMCG/Make-Your-Anchor#17]
Hi, I am now trying to build my dataset for fine-tuning and generation. However, I do not know how to create the inpainting mask, and it seems that you haven't specified which facial segmentation model you used for it. Could you please let me know the method you used?

![image](https://github.com/user-attachments/assets/45ed4c7d-8458-481d-885a-bb7984b07de1)

**Comment by bone-11** [comment]
Face parsing model is borrowed from https://github.com/zllrunning/face-parsing.PyTorch and can be found in inference directory with get_mask.py and /res/cp/79999_iter.pt. You can process the faces with the code in inference.sh, where the pathes should be modified.
```
# face alignment
python face_alignment.py \
    --imgdir_pth $body_input_dir \
    --raw_imgdir_pth $body_save_dir \
    --results_dir $body_headcrop_root \
    --crop_size 512

# get face mask
python get_mask.py \
    --input_pth $head_dir \
    --mask_pth $head_mask_dir
```

**Comment by YuGuilliman** [comment]
thank you for your quick response!
